node-backend/app/controllers/v1/usersController.js

```diff
@@ -6,7 +6,7 @@ module.exports.findUsersByOrg = async (req, res) => {
   let userData = await jwtToken.getData(token);
   db.sequelize
     .query(
-      "select users.id,users.name,users.email,users.display_name,users.profile_image_url,users.role_id,users.hi5_quota_balance from users, roles where org_id = '" +
+      "select distinct users.id,users.first_name,users.email,users.display_name,users.profile_image_url,users.role_id,users.hi5_quota_balance from users, roles where org_id = '" +
```

Comment:
Don't need join with role since we are fetching only employees you can pass role_id directly. And don't use query